common/unsafe/src/main/java/org/apache/spark/unsafe/UnsafeAlignedOffset.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
There are a number of style problems in these changes. I'll comment on a few but you'll probably want to do a more thorough review. Then we can look at content.

This is class javadoc and needs to be before the class. This is missing a copyright header that would normally go here.